aslam_offline_calibration/kalibr/scripts/kalibr_data_import.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+folder=$1
+cd $folder
```

Comment:
I'd add some sanity checks here: check the folder exists, the tar file too, etc.